aussie_clock.py

```diff
@@ -114,6 +115,25 @@ async def goob_schedule_upd():
                         await channel.send(schedule_message)
 
 
+async def birthday_check():
+    birthdays = json.loads(open("birthdays.json", "r").read())
```

Comment:
Consider using the [`aiofiles` library](https://pypi.org/project/aiofiles/) to read this data asynchronously

Reply:
that will probably something i add in later on as that would require learning a new library, and im just now learning the json one